### PR TITLE
Tic-Tac-Toe game using java

### LINKED ISSUE #289: Tic-tac-toe game using Java
## 🚀 Feature

(A clear and concise description of what the feature is.)
Tick tack toe game using Java

## Motivation
it is a beginner project which can help in understanding java better.
(Please outline the motivation for the proposal.)

## Pitch
It uses random method.
it is executed in terminal. 
it can be helpful for the beginners who want to understand java with practical knowledge.
(Please explain why this feature should be implemented and how it would be used.)

### PR DESCRIPTION
**Pull request by nitishkumar07**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?
 YES

### Description
I am a GSSOC'22 participant
Added Tic-Tac-Toe game 
(Write your answer here.)

Fixes #289
 

### Checklist

- [ X ] I've read the contribution guidelines.
- [ X ] I've checked the issue list before deciding what to submit.
- [ X ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by photon149** [comment]
@nitishkumar07 please follow PR guidelines.
- Update the issue number correctly in the PR , write the issue number in place of "#issue_no"
- Add further Details in your README , it's too short.

**Comment by nitishkumar07** [comment]
I have updated readme file 
is it ok?

**Review by photon149 — APPROVED** [review]
Looks good to me.
@prakhar3062 over to you.

**Review by prakhar3062 — APPROVED** [review]
Everything works fine

File: Desktop Application/Basic/Java/TicTacToe/.classpath [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Java/TicTacToe/.classpath:1]: Delete this file if not required !

File: Desktop Application/Basic/Java/TicTacToe/.project [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Java/TicTacToe/.project:1]: @nitishkumar07 Delete this file if not required !!

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@nitishkumar07 Delete all (.) file if not required .

**Comment by nitishkumar07** [comment]
I have removed hidden files .
Please review it.

**Review by Kushal997-das — APPROVED** [review]

**Pull request merged**